temapi/fetcher.py

```diff
@@ -0,0 +1,77 @@
+import re
+
+import requests
+import json
+from pathlib import Path
+from bs4 import BeautifulSoup
```

Comment:
bs4 is not being used anymore right? Should be removed from here and from the dependencies